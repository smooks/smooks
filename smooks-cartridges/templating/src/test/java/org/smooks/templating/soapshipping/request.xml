<root note="this node is not relevant to the test">
	<trackingNumber>
		<shipperID>Speedy</shipperID>
		<shipmentNumber>
			ZX5-20030294-731
		</shipmentNumber>
	</trackingNumber>
</root>