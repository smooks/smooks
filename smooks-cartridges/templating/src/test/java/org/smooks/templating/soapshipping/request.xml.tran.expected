<root note="this node is not relevant to the test">
	<trackingNumber>
 		Speedy:ZX5-20030294-731
	</trackingNumber>
</root>